File: LeetCode1684.java
class Solution {
    public int countConsistentStrings(String allowed, String[] words) 
    {
        int iResult = 0;

        for(int iCnt = 0 ;iCnt < words.length ;iCnt++)
        {
            for(int i = 0 ;i < allowed.length() ;i++)
            {
                char ch = allowed.charAt(i);
                String sTemp = Character.toString(ch);

                words[iCnt] = words[iCnt].replace(sTemp, "");
            }
            //System.out.println(words[iCnt] + "  " + words[iCnt].length());
            if(words[iCnt].length() == 0)
            {
                iResult++;
            }
        }    

        return iResult;
    }
}

class LeetCode1684
{
    public static void main(String[] args) 
    {
        Solution sobj = new Solution();    

        String sData[] = new String[]{"ad","bd","aaab","baa","badab"};
        int iRet = sobj.countConsistentStrings("ab", sData);

        System.out.println("Number of Consistent Strings are : " + iRet);
    }
}

File: LeetCode1684.py
class Solution:
    def RemoveAllOccurances(self ,sstr ,ch):
        sChar = set()

        for i in range(0 ,len(sstr) ,1):
            sChar.add(sstr[i])
        
        #print(sChar)

        if ch in sChar:
            sChar.remove(ch)

        #print(sChar)
        sstr = str(sChar)

        return sstr
    

    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        Result = 0
        sobj = Solution()

        for Cnt in range(0 ,len(words) ,1):
            sTemp = ""
            for i in range(0 ,len(allowed) ,1):
                sTemp = sobj.RemoveAllOccurances(words[Cnt] ,allowed[i])
            
            print(sTemp)
            
            if(len(sTemp) == 0):
                Result = Result + 1

        return Result


def main():
    sobj = Solution()
    Ret = sobj.RemoveAllOccurances("apple" ,"p")
    print(Ret)

if(__name__ == "__main__"):
    main()